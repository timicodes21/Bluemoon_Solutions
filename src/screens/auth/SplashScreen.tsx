import { View, Text, ActivityIndicator } from "react-native";
import React from "react";
import { splashStyles as styles } from "../../styles/auth/splashStyles";
import { useSplash } from "../../hooks/auth/useSplash";

const SplashScreen = () => {
  useSplash();
  return (
    <View style={styles.container}>
      <View>
        <ActivityIndicator size="large" color="#9C2CF3" />
      </View>
    </View>
  );
};

export default SplashScreen;
