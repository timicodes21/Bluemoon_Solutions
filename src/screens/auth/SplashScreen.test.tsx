import { render, screen } from "@testing-library/react-native";
import SplashScreen from "./SplashScreen";

describe("SplashScreen", () => {
  it("matches the snapshot", () => {
    const { toJSON } = render(<SplashScreen />);
    expect(toJSON()).toMatchSnapshot();
  });
});
