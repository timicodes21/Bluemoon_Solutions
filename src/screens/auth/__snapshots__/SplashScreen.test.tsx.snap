// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplashScreen matches the snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#F2F5FF",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 70,
    }
  }
>
  <View>
    <ActivityIndicator
      color="#9C2CF3"
      size="large"
    />
  </View>
</View>
`;
